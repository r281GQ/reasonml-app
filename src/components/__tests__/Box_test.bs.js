// Generated by BUCKLESCRIPT VERSION 5.0.4, PLEASE EDIT WITH CARE
'use strict';

var Jest = require("@glennsl/bs-jest/src/jest.js");
var React = require("react");
var ReactTestingLibrary = require("bs-react-testing-library/src/ReactTestingLibrary.bs.js");
var Box$ReactHooksTemplate = require("../Box.bs.js");

Jest.test("<App />", (function (param) {
        var eta = React.createElement(Box$ReactHooksTemplate.make, {
              p: /* `sc */[
                25744,
                9
              ],
              children: "from box"
            });
        var result = ReactTestingLibrary.render(undefined, undefined, eta);
        var element = ReactTestingLibrary.getByText(/* `Str */[
              4153489,
              "from box"
            ], undefined, result);
        return Jest.Expect[/* toMatchSnapshot */16](Jest.Expect[/* expect */0](element));
      }));

/*  Not a pure module */
