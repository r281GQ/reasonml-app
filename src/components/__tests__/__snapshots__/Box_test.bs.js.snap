// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> 1`] = `
<div
  class="css-nk32ej"
>
  from box
</div>
`;
